src/query/dump.cpp

```diff
@@ -536,6 +552,25 @@ PullPlanDump::PullChunk PullPlanDump::CreateInternalIndexCleanupPullChunk() {
   };
 }
 
-void DumpDatabaseToCypherQueries(query::DbAccessor *dba, AnyStream *stream) { PullPlanDump(dba).Pull(stream, {}); }
+PullPlanDump::PullChunk PullPlanDump::CreateTriggersPullChunk() {
+  return [this](AnyStream *stream, std::optional<int>) {
+    if (trigger_store_->HasTriggers()) {
+      auto triggers = trigger_store_->GetTriggerInfo();
+      for (auto &trigger : triggers) {
+        std::ostringstream os;
+        std::replace(trigger.statement.begin(), trigger.statement.end(), '\n', ' ');
```

Comment:
I think this should be copied not changed in place since you then lose information about the original statement. Not a problem currently but could be for a future me or you who don't know this small caveat